### ISSUE TITLE
Latest version broke some of queries

### ISSUE THREAD

**Issue by kak-tus** [Vertamedia/clickhouse-grafana#42]
May be it was broken in 1.4.1 or in 1.4.0.

Plugin got query and send it to Clickhouse without some symbols.
See resulting query. There is no comma here "AS t     toRelativeMinuteNum"

Settings
![image](https://user-images.githubusercontent.com/3152421/37559820-8611490a-2a3d-11e8-94dc-437be21e40bf.png)

Query
```
SELECT
    concat( smsc_id, ' (', server, ')' ) AS t,
    toRelativeMinuteNum(now()) - toRelativeMinuteNum(max(dt)) AS c
FROM $table
WHERE
    dt_part >= toDate(now()) - 1
    AND dt >= now() - 86400 * 1
    AND status = 'online'
GROUP BY t
HAVING c > 10
UNION ALL
SELECT
  concat( smsc_id, ' (', server, ')' ) AS t,
  toRelativeMinuteNum(now()) - toRelativeMinuteNum(min(dt)) AS c 
FROM $table WHERE dt_part >= toDate(now()) - 7 AND dt >= now() - 86400 * 7 AND smsc_id NOT IN (
    SELECT smsc_id
    FROM $table
    WHERE
        dt_part >= toDate(now()) - 7
        AND dt >= now() - 86400 * 7
        AND status = 'online'
    GROUP BY smsc_id)
    GROUP BY t
```

Resulting query
```
SELECT     concat(smsc_id, ' (', server, ')') AS t,     toRelativeMinuteNum(now()) - toRelativeMinuteNum(max(dt)) AS c FROM gate.kannel_status WHERE     dt_part >= toDate(now()) - 1     AND dt >= now() - 86400 * 1     AND status = 'online' GROUP BY t HAVING c > 10 UNION ALL     SELECT concat(smsc_id, ' (', server, ')') AS t     toRelativeMinuteNum(now()) - toRelativeMinuteNum(min(dt)) AS c FROM gate.kannel_status WHERE dt_part >= toDate(now()) - 7 AND dt >= now() - 86400 * 7 AND smsc_id NOT IN (     SELECT smsc_id     FROM gate.kannel_status     WHERE         dt_part >= toDate(now()) - 7         AND dt >= now() - 86400 * 7         AND status = 'online'     GROUP BY smsc_id)     GROUP BY t
```

Target table
```
CREATE TABLE gate.kannel_status ( dt_part Date,  dt DateTime,  server String,  smsc_id String,  status Enum8(\'online\' = 1, \'connecting\' = 2, \'reconnecting\' = 3, \'dead\' = 4)) ENGINE = MergeTree(dt_part, dt, 8192)
```

**Comment by kak-tus** [comment]
Sorry, I forgot: grafana was also updated from 4.6.3 to 5.0.3.

**Comment by hagen1778** [comment]
Hi @kak-tus 
There was an issue with `UNION ALL` support. I hope it fixed now in #43 
Can you verify?

**Comment by kak-tus** [comment]
Thank you, it's fine now.

I've tried broken query - it's ok.
And for test query like this
```
select 1,2,3
union all
select 4,5,6
union all
select 7,8,9
```
It's also ok.
